<html>
<head>
<style>
img {
	margin-right:10px;
}
body {
	margin-left:20%;
	margin-right:20%;
}
</style>

<link rel="stylesheet" type="text/css" href="layout/styles/style.css" />
</head>

<body>
<br>
<?php #require('login.php');?>

<!--
<hr>
<br>
-->

<h6>C1</h6>
<br>
<table>
  <tr>
	<td>
	<p>Fluidigm C1&trade; system allows users to rapidly capture and isolate up to 96 individual cells in one reaction by processing with C1&trade; Integrated Fluidic Circuits (IFCs) based on three cell sizes – small cells (5–10 &micro;m), medium cells (10–17 &micro;m), and large cells (17–25 &micro;m). This service also provides accurate and sensitive whole-transcriptome profiling of single cells, in addition to viewing multiple cell types in one chip when different CellTracker dyes are used.<br><br>
	Each C1 run will take approximately 2 to 3 days including: capturing cells, staining for viability, imaging cells, lysing cells, performing reverse transcription, long-distance PCR amplification, harvesting the amplified cDNA and generation of sequencing-ready cDNA libraries.<br>A summary of this experimental scheme is shown below.</p>
	<br><br>
	<center><img src="layout/c1pipeline.jpg"></center>
	<br>
	<p>Referring to Figure 1, cDNA synthesis is performed on C1 by using the Clontech SMARTer&reg; Ultra&reg; Low RNA Kit (Cat# 634936). Target mRNA (with PolyA+ tail) from captured cells is reverse transcribed into full-length cDNA for universal amplification. Harvest cDNA amplicons can then be further quantified on the Agilent&reg; Bioanalyzer. If desired, library sequencing can be done by the Illumina HiSeq system.</p>
	<br><br>
	<center><img src="layout/c1chemistry.jpg" style="width:750"></center>
	<span style="color:gray;font-size:12px"><br>Figure 1 – An overview of chemistry in SMARTer&reg; PCR cDNA synthesis</span><br>
	<br>
	<p>Please refer to SMARTer&reg; Ultra&reg; Low RNA Kit for Illumina&reg; Sequencing User Manual for more information (URL link: <a href="http://www.clontech.com/xxclt_ibcGetAttachment.jsp?cItemId=41709">Click Here</a>).</p>
	<br>
	<p>List of reagents required for this experiment:</p>
	<br><br>
	<p>1. C1&trade; IFC <span style="color:red">(check out by user)</span><br>
	2. C1&trade; Single Cell Auto Prep Reagent Kit for mRNA seq<br>
	3. SMARTer&reg; Ultra&trade; Low RNA Kit for Illumina Sequencing<br>
	4. Quant-IT&trade; PicoGreen&reg; dsDNA Assay Kit<br>
	5. Nextera XT DNA Sample Prep Kit<br>
	6. Nextera XT DNA Sample Preparation Index Kit<br>
	7. Agencourt AMPure XP DNA Binding Beads</p>

	<span style="color:gray;font-size:12px"><br>*Note: Items 2 to 7 are provided by the core.</span><br><br>
	<p>If you are interested in reading more details about this technology, kindly refer to the following publications,<br><br>
	1. Mora-Castilla S, To C, Vaezeslami S <i>et al</i>. (2016) Miniaturization Technologies for Efficient Single-Cell Library Preparation for Next-Generation Sequencing. <i>J Lab Autom.</i> 21(4): 557-67.<br>
	2. Wu AR, Neff NF, Kalisky T <i>et al</i>. (2014) Quantitative assessment of single-cell RNA-sequencing methods. <i>Nat Methods.</i> 11(1):41-46.</p>
	<br><br>
	<hr>
	<br>
	<h6>SAMPLE PREPARATION<h6><br>
	<h6>A. PREPARED BY USERS<h6>
	<br>
	<p>Prior to starting the experiment, users should harvest, wash, and count cells.
	<br><br>
	<ul>
	<li>Cells need to be freshly dissociated into single-cell suspension for C1 run, please kindly notify a responsible person at least three days before the time of cell harvest</li>
	<li>It is also important to check cells under a microscope after cell counting to ensure cells are dispersed</li>
	<li>He/she should fill a submission form about the number of runs (max. of two cell lines/run) to be performed and other respective information</li>
	<li>The service will only be provided after all the details have been acknowledged by the core</li>
	</ul>
	<br>
	<p>All samples should meet the following criteria for submission:<br><br>
	1. Cells are required to be dissociated into single cell suspension<br>
	2. Please check out from the core website, the appropriate C1 IFC regarding to your sample cell size<br>
	3. If you are wish to perform two cell lines per chip, please make sure the two cell sizes are within the same size range<br>
	4. Submit at least 500 &micro;L of 3.0 x 10<sup>5</sup> cells/mL for each sample*</p><br><br><br>
	
	<table align="center" cellpadding="0" cellspacing="0" border="0">
	  <tr align="center">
		<td rowspan="2" valign="middle"><i>Volume of additional media =&nbsp;&nbsp;&nbsp;</i></td>
		<td style="border-bottom:solid;	border-bottom-width:medium"><i>conc. of cells</i></td>
		<td valign="middle">&nbsp;&nbsp;- 1</td>
	  </tr>
	  <tr>
		<td align="center"> 3.0 x 10<sup>5</sup></td>
	  </tr>
	</table>
	<br>
	<hr>
	<br>
	<h6>B. ASSISTED BY CORE<h6>
	<br>
	<p>Users will be assisted by a core personnel during the whole course of the work flow on mRNA sequencing preparation and library preparation for sequencing on HiSeq.
	<br>
	<br>The protocol is adapted from the 'Fluidigm using C1 to Generate Single-Cell cDNA Libraries for mRNA Sequencing' Protocol <a target="_blank" class="button" href="https://www.fluidigm.com/binaries/content/documents/fluidigm/resources/c1-mrna%E2%80%90seq-pr-100%E2%80%907168/c1-mrna%E2%80%90seq-pr-100%E2%80%907168/fluidigm%3Afile">Click</a>
	<br><br>
	
	</td>
   </tr>
</table>
</body>
</html>



